### DISCUSSION TITLE
Change crops names

### DISCUSSION THREAD

**Discussion by scohill** [ultralytics/ultralytics#4856] [Q&A] [answered]
is there a way to change the crops names from name2,name3,.. to name-2 name-3

**Comment by scohill** [comment, marked as answer]
for anyone trying to do this someone answered me just change the sep internal argument in files.py in ultralytics

**Reply by glenn-jocher** [reply]
@scohill great to hear that you've found a solution! If you have any more questions or need further assistance with YOLOv8, feel free to reach out. Happy coding! 😊👍
